tests/baselines/reference/nodeModules1(module=nodenext).errors.txt

```diff
@@ -191,10 +191,10 @@ tests/cases/conformance/node/index.ts(84,21): error TS2835: Relative import path
     // These should _mostly_ work - `import = require` always desugars to require calls, which do have extension and index resolution (but can't load anything that resolves to esm!)
     import m24 = require("./");
                          ~~~~
-!!! error TS1471: Module './' cannot be imported using this construct. The specifier only resolves to an ES module, which cannot be imported synchronously. Use dynamic import instead.
+!!! error TS1471: Module './' cannot be imported using this construct. The specifier only resolves to an ES module, which cannot be imported with 'require'. Use an ECMAScript import instead.

```

Comment:
It took me a bit to convince myself that this worked because I forgot that we allowed `import = require` in ES modules.

Reply:
Prediction: no user will ever see this error message